bf2fc6cc711aee1a0c2a/kas-fleet-manager
# ci: do not copy the bin folder when creating the docker image for CI run

## Description


During local testing of the CI containers files, it was observed that sometimes the pr_check.sh run fails because of existing local bin files being copied over to the CI container.
This ensure that the copied local bin files are deleted and let our make setups download the needed dependencies.
 
## Verification Steps


Verifying it requires a bit of a dance. 
@JameelB has the setup already in place due to https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1279 and will perform the verification steps that the CI container run as expected.

## Checklist (Definition of Done)
